package.json

```diff
@@ -90,12 +90,12 @@
 		"@wordpress/block-serialization-default-parser": "4.35.1",
 		"@wordpress/blocks": "12.12.1",
 		"@wordpress/commands": "0.6.3",
-		"@wordpress/components": "25.1.3",
+		"@wordpress/components": "23.9.0",
```

Comment:
This seems wrong?

Reply:
Yeah, I made a mistake, see https://github.com/WordPress/wordpress-develop/pull/4748#issuecomment-1612157261. Should be fixed now.